radioapp/ui/ui_backlight.py

```diff
@@ -0,0 +1,62 @@
+from tkinter import *
+
+
+class UIBackLight():
+	"""Creates UI Backlight Screen."""
+
+	def __init__(self):
+		"""Creates a window, and adds slider and label for UI Backlight."""
+
+		self.window = Tk()
+		self.window.title("Backlight Setting")
+		self.window.geometry("160x128")
+		self.window.configure(background = "#ff9966")
+		self.window.resizable(0, 0)
+		self.window.pack_propagate(0)
+		
+		#Binds key presses to change slider value.
+		self.window.bind("<Up>", self.up_key)
+		self.window.bind("<Down>", self.down_key)		
+		
+		#Adds slider to the window.
+		self.add_slider()
+		
+		#Adds label to the window.
+		self.add_label()		
+		
+		self.window.mainloop()
+		
+	def add_slider(self):

```

Comment:
When I ran this, the slider was vertical, making it overlap the text "djust Brightness"

Reply:
Hmm,  I'm using it on windows and in my machine it's not overlapping the Adjust Brightness. It's looks fine here.

Reply:
A circle with a percentage in the center would be super slick.